snippets/array-pull-(mutates-array).md

```diff
@@ -1,21 +1,15 @@
-### Pull
+### Array pull (mutates array)
 
 Use `Array.filter()` to pull out the values that are not needed. 
 Use `Array.length = 0` to mutate the passed in array by resetting it's length to zero.
 Use `Array.push()` to re-populate the original array to equal the pulled values
 
 ```js
 const pull = (arr, ...args) => {
-  let pulled = arr.filter((v, i) => args.indexOf(v) === -1);
-  arr.length = 0;
-  pulled.forEach(v => arr.push(v));
-}
-
+  let pulled = arr.filter((v, i) => args.includes(v));
```

Comment:
It's a good improvement, however there have to be `!args.includes(v)`.
I've made PR with this fix - https://github.com/Chalarangelo/30-seconds-of-code/pull/213